app/src/main/java/io/gnosis/safe/helpers/ConnectivityInfoProvider.kt

```diff
@@ -12,12 +12,12 @@ class ConnectivityInfoProvider(private val connectivityManager: ConnectivityMana
     var offline: Boolean = true
 
     private val networkCallback = object : ConnectivityManager.NetworkCallback() {
-        override fun onAvailable(network: Network?) {
+        override fun onAvailable(network: Network) {
```

Comment:
This fixes another bug, right? Or was this not meant to be in this PR?

Reply:
no, it was fixed in previous pr with try catch around read-only network request